### ISSUE TITLE
Unable to Re-Deploying Manifests after change

### ISSUE THREAD

**Issue by dorsany** [rancher/k3s#737]
**Describe the bug** and **How To Reproduce**
I have a k3s installed using the following installation command:
```curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION='v0.6.1' sh -s -  --docker --no-deploy traefik```
I am also using  [Auto-Deploying Manifests](https://rancher.com/docs/k3s/latest/en/configuration/#auto-deploying-manifests) to deploy my ```nginx-ingress``` with this manifest:
```
cat > /var/lib/rancher/k3s/server/manifests/ingress-nginx.yaml <<- EOF
apiVersion: helm.cattle.io/v1
kind: HelmChart
metadata:
  name: ingress-nginx
  namespace: kube-system
spec:
  chart: stable/nginx-ingress
  version: 1.7.0
  targetNamespace: ingress-nginx
  valuesContent: |-
    tcp:
      15671: "default/rabbitmq:15671"
      27017: "default/mongodb:27017"
EOF
```
After the installation, the k3s deployed my nginx-ingress.
The thing is that After i tried to change one of the values (like add more port) in my manifest file ```/var/lib/rancher/k3s/server/manifests/ingress-nginx.yaml``` the k3s didn't Re-Deploying that Manifest. even after trying to restart the k3s service with ```service k3s restart``` or to delete and recreate the file or to rename the file.
The only way to achieve Re-Deploying that Manifest was to uninstall the k3s using ```/usr/local/bin/k3s-uninstall.sh``` and to re-install the k3s with the above command, and I prefer to perform upgrade and not to delete and install.

**Expected behavior**
after any file **changed**, the k3s shold re deploy the manifest or to get a command to re-deploy all of the manifests ubder ```/var/lib/rancher/k3s/server/manifests``` like ```helm upgrade```

related: https://github.com/rancher/k3s/issues/187

**Comment by warmchang** [comment]
`kubectl delete helmcharts ingress-nginx -n kube-system`

and then `systemctl restart k3s.service`

**Comment by dorsany** [comment]
> `kubectl delete helmcharts ingress-nginx -n kube-system`
> 
> and then `systemctl restart k3s.service`

Thanks @warmchang for your answer, this solution is working, but with that solution when I'm deleteing my helm charts objects under ```kubectl get helmcharts.helm.cattle.io -n kube-system``` it is also removes my services (the deployments, configmaps and etc.. in that case) just like ```helm delete```

no way just to perform ```helm upgrade```?

**Comment by erikwilson** [comment]
Should be fixed in #742 and available shortly in v0.8.1. Please re-open if this is still an issue.
